apis/v1alpha1/local_object_reference_types.go

```diff
@@ -35,9 +35,9 @@ type ServicesDefaultLocalObjectReference struct {
 	//   resource: foos
 	//   name: myfoo
 	//
-	// +optional
+	// +kubebuilder:validation:MaxLength=253
```

Comment:
// +kubebuilder:validation:MinLength=1

Reply:
Updated, thanks!

Reply:
I think I may have been wrong here. I'm not sure it makes sense to enforce min on a field that is being defaulted.